x-pack/legacy/plugins/siem/public/components/ml/api/anomalies_table_data.ts

```diff
@@ -25,10 +23,9 @@ export interface Body {
 
 export const anomaliesTableData = async (
   body: Body,
-  headers: Record<string, string>,
+  kbnVersion: string,
   signal: AbortSignal
 ): Promise<Anomalies> => {
-  const [kbnVersion] = useKibanaUiSetting(DEFAULT_KBN_VERSION);
```

Comment:
The first rule of Hooks Club is [_Don’t call Hooks from regular JavaScript functions_](https://reactjs.org/docs/hooks-rules.html#only-call-hooks-from-react-functions)...

We've got `eslint-plugin-react-hooks` -- any idea why wasn't this caught?

Reply:
nope no idea

Reply:
maybe because `useKibanaUISetting` is not ordinary hook

Reply:
oh I know why it was working before because we were importing the npStart to get the uiSettings but now we are using the useKibanaCore and useKibanaPlugins hooks so that's why this error came up